cg/meta/workflow/balsamic.py

```diff
@@ -469,7 +469,7 @@ def run_analysis(
             {
                 "--account": self.account,
                 "--mail-user": self.email,
-                "--qos": priority or self.get_priority_for_case(case_id=case_id),
+                "--qos": priority or self.get_slurm_qos_for_case(case_id=case_id),
```

Comment:
same, we could also change the variable `priority` to `qos`

Reply:
Done